web/templates/settings/alert_word_settings.hbs

```diff
@@ -1,18 +1,18 @@
 <div id="alert-word-settings" class="settings-section" data-name="alert-words">
     <form id="alert_word_info_box">
         <p class="alert-word-settings-note">
-            {{t "Alert words allow you to be notified as if you were @-mentioned when certain words or phrases are used in Zulip. Alert words are not case sensitive."}}
+            {{t "Watched phrases allow you to be notified as if you were @-mentioned when certain words or phrases are used in Zulip. Watched phrases are not case sensitive."}}
```

Comment:
This could also be improved in that prep commit ...

`Alert words allow you to receive desktop notifications as if you were @-mentioned when ...`